addons/payment_adyen/controllers/main.py

```diff
@@ -263,11 +263,17 @@ def adyen_notification(self):
                     notification_data['resultCode'] = 'Authorised' if success else 'Error'
                 else:
                     continue  # Don't handle unsupported event codes and failed events
-
-                # Handle the notification data as a regular feedback
-                PaymentTransaction.sudo()._handle_feedback_data('adyen', notification_data)
-            except ValidationError:  # Acknowledge the notification to avoid getting spammed
-                _logger.exception("unable to handle the notification data; skipping to acknowledge")
+            except ValidationError:
+                # Avoid noise in the logs when the (pos, payment)_adyen are both installed
+                _logger.warning("unable to find the transaction; skipping to acknowledge")
+            else:
```

Comment:
>> Hello duck Nice use of the try/except/else! Maybe slide L248-L265 into the else too then, for clarity purpose? These lines won't raise a ValidationError anyway thinking (It's a suggestion, do as you prefer here ^^)
>
> I prefer to have smaller diff :)

I too would prefer if the non-relevant part were moved into the `else`. Not (only) because I find it better looking but because it makes the final code easier to read: only the methods that raise a `ValidationError` is surrounded in `try/except` clauses, and the rest of the code can leave its life outside of them.

I totally understand the motivation to limit the size of the diff, in particular in a fix, but, in general, I believe it's more important to have a clear code than a clear diff, as the former is read way more often than the latter. If it comes without sacrificing the clarity, then sure you should aim for a smaller diff ;)

That said, I don't think you'll have a much larger diff if you move the `except/else` because the rest of the code is already correctly indented.